yarn/pom.xml

```diff
@@ -102,6 +102,10 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-servlet</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlets</artifactId>
+    </dependency>
```

Comment:
Looks good to me. Just confirming there really are both "jetty-servlet" and "jetty-servlets" artifacts that we need both of?